charts/steward/README.md

```diff
@@ -0,0 +1,138 @@
+# Steward Helm Chart
+
+Install and configure [Steward][] on Kubernetes.
+
+## Table of Content
+
+- [Steward Helm Chart](#steward-helm-chart)
+  - [Table of Content](#table-of-content)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Upgrade](#upgrade)
+  - [Uninstall](#uninstall)
+  - [Chart Configuration](#chart-configuration)
+  - [Custom Resource Definitions](#custom-resource-definitions)
+
+## Prerequisites
+
+This Helm chart requires _Helm 3_ or higher.
+
+The Steward Helm chart is currently not published in any public Helm repository.
+Therefore it must be installed from a source checkout.
+
+## Install
+
+Use the `helm install` command to install the Steward Helm chart:
+
+```bash
+helm install RELEASE_NAME CHECKOUT_DIR/charts/steward/ ...
+```
+
+The `helm install` command has a parameter `--namespace` that defines the target namespace of the release.
+Normally this is the namespace where the application will be installed to.
+Helm also stores the release data in that namespace.
+However, the Steward chart does not use the release namespace but has a separate parameter `targetNamespace.name` defining the namespace where Steward will be installed to.
+This allows to include the Steward chart as dependency into another chart but still install into an own namespace.
+The Helm release target namespace and the Steward target namespace can be equal if required.
+
+If the Steward target namespace deliberately exists already, parameter `targetNamespace.create` should be set to `false` to suppress a resource conflict error.
+
+Do not use the `--no-hooks` option of the `helm install` command.
+Hooks are required for a consistent installation.
+
+## Upgrade
+
+Use the `helm upgrade` command to upgrade Steward releases:
+
+```bash
+helm upgrade RELEASE_NAME CHECKOUT_DIR/charts/steward/ ...
+```
+
+To reuse values from the current release revision, __do _NOT_ use the `--reuse-values` option__ of the `helm upgrade` command.
+This option will not only reuse overridden values, but also the built-in values of the current release's chart version.
+The result might be unexpected. Instead:
+
+1.  Retrieve only the overridden values from the current release:
```

Comment:
This is also the way [GitLab recommends](https://docs.gitlab.com/charts/installation/upgrade.html) for their Helm chart.

The process ensures that the Helm release contains the correct distinction between overridden values and those set from the respective Chart version's `values.yaml`.

In case the overridden values are kept/versioned separately (e.g. in Git), the step to retrieve them from the Helm release can be skipped.

So I would keep the description.